README.md

```diff
@@ -355,10 +350,10 @@ You must set this variable to pass variables to Linux builds (since they execute
 
 You can use `$PATH` syntax to insert other variables, or the `$(pwd)` syntax to insert the output of other shell commands.
 
-Example: `CFLAGS="-g -Wall" CXXFLAGS="-Wall"`  
-Example: `PATH=$PATH:/usr/local/bin`  
-Example: `BUILD_TIME="$(date)"`  
-Example: `PIP_EXTRA_INDEX_URL="https://pypi.myorg.com/simple"`  
+Example: `CFLAGS="-g -Wall" CXXFLAGS="-Wall"`
```

Comment:
See https://gist.github.com/shaunlebron/746476e6e7a4d698b373
(Which is also weird, because https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#lines claims something else.)

Reply:
I now replaced trailing `  `  by `\` to show the visual intent clearly in the file and deal with editors which just strip trailing spaces (as do mine, but I could reconfigure for markdown).

Reply:
Ah, yes, that might clear out future confusion as well :)
Still wondering if we should make that a separate commit/PR outside of this PR, though? It keeps the diff smaller. Then again, it's only in REAME, so it's maneagable, actually.

Reply:
I can make the change in a separate commit if this is clearer but it will be in the PR.

Reply:
Don't bother too much about it. I guess it's fine.